pkg/util/rand/rand.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package rand provides utilities related to randomization.
+package rand
+
+import "math/rand"
+
+var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
+
```

Comment:
// Consider adding something as listed below
// refer to https://golang.org/pkg/math/rand/#Seed
// Without the seeding, we are not be guaranteed to get different number sequences.
// Doing the seeding in init, helps it be a one-time initialization. http://golang.org/doc/effective_go.html#init

import "time"
func init() {
  rand.Seed(time.Now().UTC().UnixNano())
}